MetaMask/metamask-extension
# MV3: Re-activate service worker and reconnect UI streams

Fixes: https://github.com/MetaMask/metamask-extension/issues/14559

1. reactivate in-active service worker
2. re-connect UI stream on full-page implementation of extension (this still needs to be fixed for extension popup)

How to test:
1.  Use MV3 via `yarn start:mv3`
2. Open home screen wait a five minutes, where service worker should disconnect, and keep using the extension -- it should work
3. Open DevTools in the UI 
4. Choose Application -> Service Workers and kill the service worker
5. Notice that the service worker restarts
